daemon/src/position_metrics.rs

```diff
@@ -8,24 +10,29 @@ use model::Position;
 use model::Usd;
 use rust_decimal::prelude::FromPrimitive;
 use rust_decimal::Decimal;
+use std::collections::HashMap;
 use std::time::Duration;
-use tokio_tasks::Tasks;
 use xtra_productivity::xtra_productivity;
-use xtras::SendInterval;
+use xtras::SendAsyncSafe;
 
 // TODO: ideally this would be more often
 pub const UPDATE_METRIC_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
This should now be unused right? If it would have been private from the beginning, rustc would probably flag it as `dead_code` now!

Reply:
The TODO is also stale I believe!